tests/util.py

```diff
@@ -54,8 +54,17 @@ def wrapper(self):
     return wrapper
 
 
-def remove_space(str):
-    return str.replace(' ', '').replace('\n', '')
+def _remove_control_sequence(string):
+    return re.sub(r'\x1b[^m]+m', '', string)
+
+
+def _remove_whitespace(string):
+    return re.sub(r'\s', '', string)
+
+
+def assert_in_multi_line(sub_string, string):
+    # assert sub_string is in string, with all whitespaces, line breaks and control sequences ignored
+    assert _remove_whitespace(sub_string) in _remove_control_sequence(_remove_whitespace(string))
```

Comment:
This function does the job of removing control sequences which are previously removed by colorama.